docs/name-owners/atomic-name-trading/index.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: page
+title: Atomic Name Trading
+---
+
+{::options parse_block_html="true" /}
+
+Because both currency transactions and name operations share a common technical backing (the Namecoin blockchain), it is possible to construct transactions that transfer both namecoins and a name _atomically_. This means that it is guaranteed that either both transfers succeed or both transfers are not executed at all. A potentially very useful application of this technique is trading of names for namecoins, since this makes it possible to sell or buy a name in a completely trustless manner without the necessity to use escrow.
+
+At the moment, it is possible to make such transfers only using RPC commands to namecoind directly. But in the future, hopefully an easy-to-use UI will be developed that handles all the technicalities. On this page, the basic work-flow will be described with RPC commands. We will assume as an example that Bob wants to buy 'd/my-cool-domain' from Sally for 10 NMC. The work-flow for this transaction is shown below on the test net. 
+
+## Bob Prepares a Buy-Offer
+
+Bob finds the domain of interest and Sally's offer to sell the domain. This could happen, for instance, via Sally using a special value for the domain or by some other channel out of the scope of this page. For instance: 
+
+```
+$ namecoind name_show d/my-cool-domain
+{
+  "name" : "d/my-cool-domain",
+  "value" : "for sale by Sally for 10 NMC, pay to mm1mGfLEMjFPyGsbkcoYiN6yiPcBztK6Dw",
+  "txid" : "00abd8282558574369038722d24c65355e1b98ca3f00d7d09cb7bea8732b598a",
+  "address" : "mt2vTvKfvfb9TKfxB8ns7NVrY4cYzRs73b",
+  "expires_in" : 36000
+}
+```
+
+Now, Bob can prepare an atomic transaction that sends 10 NMC to Sally and transfers __d/my-cool-domain__ at the same time to himself. We assume that Sally wants the 10 NMC sent to __mm1mGfLEMjFPyGsbkcoYiN6yiPcBztK6Dw__, while Bob wants the name to be transferred to __mwKDtLZ9EHSeL8TjkZAcppsnb7M3LRhoGR__. 
+
+To construct the transaction, Bob needs to use the "createrawtransaction" RPC command. It needs three pieces of information: 
```

Comment:
The original wiki page was written at a time when `namecoin-tx` did not exist yet.  I personally think that using `namecoin-tx` is easier for work on the CLI, although of course the RPC interface is what would be used by a frontend.  So perhaps you can consider changing the commands here to use `namecoin-tx` instead of RPCs where possible (i.e. for creating and decoding the raw transaction).  I have no strong opinion on this, though, so feel free to keep it as is.

Reply:
IMO both the RPC and `namecoin-tx` approaches are useful to document.  I'd be happy to take a PR that adds `namecoin-tx` documentation as well, but I'd prefer not to replace the RPC docs here.

Reply:
Makes sense.